services/core-web/src/components/Forms/MineRecordForm.js

```diff
@@ -506,6 +507,30 @@ export class MineRecordForm extends Component {
             <FieldArray name="mine_types" component={renderTypeSelect} />
           </>
         )}
+        <Row gutter={16}>
+          <Col md={12} xs={24}>
+            <Form.Item>
+              <Field
+                id="number_of_mine_employees"
+                name="number_of_mine_employees"
+                label="Number of Mine Employees"
+                component={renderConfig.FIELD}
+                validate={[integer, required]}
+              />
```

Comment:
I don't think we can make this required. Lets check with Aaron. 

It may not be data that ministry staff have, and thus will prevent them from editing anything else on this form.

Reply:
Fixed. Thanks :)